cmd/benchdb/README.md

```diff
@@ -0,0 +1,59 @@
+## BenchDB
+
+benchdb is a command line tool to test the performance of TiDB.
+
+### QuickStart
+
+Make sure you have started PD and TiKV, then run:
+
+```
+    ./benchdb -run="create,truncate,insert:0_10000,update-random:0_10000:100000,select:0_10000:10"
+```
+
+
+### Arguments
+
+#### `run`
+The `run` argument defines the workflow of the test, you can define
+multiple jobs, separated by `,`, the jobs are executed sequentially.
+ 
+* `create` creates a table, currently it's just a typical simple table, with a few columns.
+
+* `truncate` truncates the table.
+
+* `insert:xxx_yyy` inserts rows starts with ID in `[xxx, yyy)`.
+ e.g. `insert:0_10000` inserts 10000 rows with id in range  `[0, 9999)`.
+   
+* `update-random:xxx_yyy:zzz` updates a row with a random ID in range `[xxx, yyy)`, do `zzz` times.
+ e.g. `update-random:100_200:50` updates 50 random rows with ID in range `[100, 200)`.
+ 
+* `update-range:xxx_yyy:zzz` update a range of rows with ID in range `[xxx, yyy)`, do `zzz` times. 
+  
+* `select:xxx_yyy:zzz` select rows with ID range in `[xxx, yyy)`, do `zzz` times.
+
+* `gc` does a manually triggered GC, so we can compare the performance before and after GC. 
+ 
+ The output shows execution time.
+ 
+#### `table`
+
+Name of the table, so we can create many tables for different tests without the need to clean up.
+Default is `bench_db`.
+  
+#### `blob`
+
+blob column size in bytes, so we can test performance for different row size.
+Default is `1000`.
+
+#### `batch`
+
+batch number of statements in a transaction, used for insert and update-random only, to speed up the test workflow.
+Default is `100`.
+
+#### `addr`
+
+PD address. Default is `127.0.0.1:2379`.
+
+### `L`
+
+log level. Default is `warn`.
```

Comment:
The result output may change frequently.
If added it in README, we need to update README whenever we change the output.